src/braket/default_simulator/density_matrix_simulator.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import sys
+
+from braket.device_schema.simulators import (
+    GateModelSimulatorDeviceCapabilities,
+    GateModelSimulatorDeviceParameters,
+)
+
+from braket.default_simulator.density_matrix_simulation import DensityMatrixSimulation
+from braket.default_simulator.simulator import BaseLocalSimulator
+
+
+class DensityMatrixSimulator(BaseLocalSimulator):
+
+    DEVICE_ID = "braket_dm"
+
+    def initialize_simulation(self, **kwargs):
+        qubit_count = kwargs.get("qubit_count")
+        shots = kwargs.get("shots")
```

Comment:
These should be arguments of `initialize_simulation`. As well, have an unimplemented `initialize_simulation` method in `BaseLocalSimulator` that documents the method.

Reply:
But the density_matrix_simulator doesn't have batch_size

Reply:
`batch_size` can come from the **kwargs

Reply:
What I mean is, batch_size is a parameter in the initialize_simulation in BaseLocalSimulator, while here it is not.

Reply:
Oh, that's why I only highlighted qubit_count and shots.